CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Unreleased]
+
+### Breaking Changes
+
+- Remove experimental features from release builds and workflows
+
+
```

Comment:
This isn't a breaking change because the last experimental build we published was 1.8, which was 9 months ago.

```suggestion
```